webapp/README.md

```diff
@@ -33,48 +33,74 @@ $ yarn build
 $ yarn start
 ```
 
-### Storybook
+### Run tests
 
-We encourage contributors to use Storybook to test out new components in an isolated way, and benefit from its many features.
-See the docs for live examples and answers to FAQ, among other helpful information. ![Storybook docs](https://storybook.js.org/docs/basics/introduction/)
+We ensure the quality of our frontend code by using
+- [ESLint](https://eslint.org/) for checking our JavaScript code
+- [Jest](https://jestjs.io/) and [Vue Test Utils](https://vue-test-utils.vuejs.org/) to unit test our components
+- [Storybook](https://storybook.js.org/) to document and manually test our components in an isolated playground
+
+For more information see our [frontend testing guide](testing.md). Use these commands to run the tests:
 
 {% tabs %}
-{% tab title="Docker" %}
+{% tab title="With Docker" %}
 
-After you have started the application following the instructions above, in another terminal run:
+After starting the application following the above guidelines, open new terminal windows for each of these commands:
 
 ```bash
-$ docker-compose exec webapp yarn storybook
+# run eslint
+$ docker-compose exec webapp yarn lint
 ```
-The output should look similar to this:
 
-![Storybook output](../.gitbook/assets/storybook-output.png)
+```bash
+# run unit tests
+$ docker-compose exec webapp yarn test
+```
 
-Click on the link http://localhost:3002/ to open the browser to your interactive storybook.
+```bash
+# start storybook
+$ docker-compose exec webapp yarn storybook
+```
+
+You can then visit the Storybook playground on `http://localhost:3002`
 
 {% endtab %}
 
 {% tab title="Without Docker" %}
-Run the following command:
+
+After starting the application following the above guidelines, open new terminal windows and navigate to the `/webapp` directory for each of these commands:
 
 ```bash
-# in webapp/
-yarn storybook
+# run eslint in /webapp
+$ yarn lint
 ```
 
-Open http://localhost:3002/ in your browser
+```bash
+# run unit tests in /webapp
+$ yarn test
+```
 
-{% endtab %}
-{% endtabs %}
+```bash
+# start storybook in /webapp
+$ yarn storybook
+```
 
+You can then visit the Storybook playground on `http://localhost:3002`
 
+{% endtab %}
+{% endtabs %}
 
-## Styleguide
+## Styleguide Migration
 
-All reusable Components \(for example avatar\) should be done inside the [Nitro-Styleguide](https://github.com/Human-Connection/Nitro-Styleguide) repository.
+We are currently in the process of migrating our styleguide components and design tokens from the [Nitro Styleguide](https://github.com/Human-Connection/Nitro-Styleguide) into the main [Human Connection repository](https://github.com/Human-Connection/Human-Connection) and refactoring our components in the process. During this migration, our new components will live in a `view` folder to separate them from the old, yet untouched components.
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-31T10:27:47Z" title="Thursday, October 31st 2019, 11:27:47 am +01:00">Oct 31, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -33,48 +33,74 @@ $ yarn build
 $ yarn start
 ```
 
-### Storybook
+### Run tests
 
-We encourage contributors to use Storybook to test out new components in an isolated way, and benefit from its many features.
-See the docs for live examples and answers to FAQ, among other helpful information. ![Storybook docs](https://storybook.js.org/docs/basics/introduction/)
+We ensure the quality of our frontend code by using
+- [ESLint](https://eslint.org/) for checking our JavaScript code
+- [Jest](https://jestjs.io/) and [Vue Test Utils](https://vue-test-utils.vuejs.org/) to unit test our components
+- [Storybook](https://storybook.js.org/) to document and manually test our components in an isolated playground
+
+For more information see our [frontend testing guide](testing.md). Use these commands to run the tests:
 
 {% tabs %}
-{% tab title="Docker" %}
+{% tab title="With Docker" %}
 
-After you have started the application following the instructions above, in another terminal run:
+After starting the application following the above guidelines open new terminal windows for each of these commands:
 
 ```bash
-$ docker-compose exec webapp yarn storybook
+# run eslint
+$ docker-compose exec webapp yarn lint
 ```
-The output should look similar to this:
 
-![Storybook output](../.gitbook/assets/storybook-output.png)
+```bash
+# run unit tests
+$ docker-compose exec webapp yarn test
+```
 
-Click on the link http://localhost:3002/ to open the browser to your interactive storybook.
+```bash
+# start storybook
+$ docker-compose exec webapp yarn storybook
+```
+
+You can then visit the Storybook playground on `http://localhost:3002`
 
 {% endtab %}
 
 {% tab title="Without Docker" %}
-Run the following command:
+
+After starting the application following the above guidelines open new terminal windows for each of these commands:
 
 ```bash
-# in webapp/
-yarn storybook
+# run eslint
+$ yarn lint
 ```
 
-Open http://localhost:3002/ in your browser
+```bash
+# run unit tests
+$ yarn test
+```
 
-{% endtab %}
-{% endtabs %}
+```bash
+# start storybook
+$ yarn storybook
+```
 
+You can then visit the Storybook playground on `http://localhost:3002`
 
+{% endtab %}
+{% endtabs %}
 
-## Styleguide
+## Styleguide Migration
 
-All reusable Components \(for example avatar\) should be done inside the [Nitro-Styleguide](https://github.com/Human-Connection/Nitro-Styleguide) repository.
+We are currently in the process of migrating our styleguide components and design tokens from the [Nitro Styleguide](https://github.com/Human-Connection/Nitro-Styleguide) into the main [Human Connection repository](https://github.com/Human-Connection/Human-Connection) and refactoring our components in the process. During this migration our new components will live in a `view` folder to separate them from the old, yet untouched components.
```


```suggestion
We are currently in the process of migrating our styleguide components and design tokens from the [Nitro Styleguide](https://github.com/Human-Connection/Nitro-Styleguide) into the main [Human Connection repository](https://github.com/Human-Connection/Human-Connection) and refactoring our components in the process. During this migration, our new components will live in a `view` folder to separate them from the old, yet untouched components.
```